src/main/resources/META-INF/plugin.xml

```diff
@@ -58,6 +58,7 @@
 
     <change-notes><![CDATA[
       <ul>
+        <li>8.1 - 6 new rules for C# in Rider. Bug fixes, fewer FPs and improvements for many languages.</li>
```

Comment:
I think there are a few other things worth mentioning:

* Support of Kotlin 1.8
* Support for tiarmclang compiler (for this one we might want to check with the CFamily bubble if we need to do something specific for CLion, we recently had to for clang-cl)